chapter-2-the-integers/exercice-17a.md

```diff
@@ -0,0 +1,30 @@
+Let $f_n$ be the $n$ th fibbonacci number, with $f_1 = 1$ and $f_2 = 1$ Prove $f_n < 2^n$ for $n \in \mathbb{N}$
+
+**Base Cases**
+
+Let $n = 1$. then
```

Comment:
As you wish, my master.

![image](https://github.com/astute-teletubby/abstract-algebra/assets/122752868/61a5d60d-88c4-4c84-bfbd-5720ddd55b9c)